content/zh/blog/2024/link-the-istio-service-mesh-for-people-who-have-stuff-to-do.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "外部文章：为繁忙的人们准备的 Istio 服务网格"
+description: 了解 Luca Cavallin 使用 Istio 的经验。
+publishdate: 2024-10-10
+attribution: Luca Cavallin - GitHub; Translated by Wilson Wu (DaoCloud)
+link: https://wilsonwu.me/blog/2024/the-istio-service-mesh-for-people-who-have-stuff-to-do/
```

Comment:
`https://wilsonwu.me/` looks good, but is it suitable to link to a personal website?

Reply:
I think the ext-linked blog is as this way, #15770 also for the EN version is.